README.md

```diff
@@ -122,58 +127,52 @@ provider classes, and not directly call into `CommandReference` or `Node`.
 
 These utilities can be used directly on a Cisco device (as used by Puppet
 and Chef) or can run on a workstation and point to a Cisco device (as used
-by the included minitest suite).
-
-### Usage on a Cisco device
+by the included minitest suite).  The Client and Node APIs will read
+connection information (host, username, etc.) from a
+[configuration file](#configuration). When creating a Client or Node,
+you can specify the label of the node in the config file to connect to (if no
+label is specified, "default" is assumed). If a configuration file cannot be
+found, the Client or Node will attempt to connect to the local device.
 
 #### Low-level Client API
 
 ```ruby
 require 'cisco_node_utils'
 
-# get a connection to the local device
+# get a connection to the "default" node in the config file
 client = Cisco::Client.create()
 
-client.get(command: 'show version')
+# get a connection to the "nxapi_remote" node in the config file
+#client = Cisco::Client.create('nxapi_remote')
+
+# via cli
+version = client.get(command: 'show version')
 client.set(values: 'vtp domain mycompany.com')
+
+# via yang
+bgp_cfg = client.get(data_format: :yang_json, command: '{"Cisco-IOS-XR-ipv4-bgp-cfg:bgp": [null]}')
+client.set(data_format: :yang_json, 
+           values: '{"Cisco-IOS-XR-infra-rsi-cfg:vrfs":{"vrf":[{"vrf-name":"BLUE","create":[null]}]}}')
 ```
 
 #### High-level Node API
 
 ```ruby
 require 'cisco_node_utils'
 
-# get a connection to the local device
+# get a connection to the "default" node in the config file
 node = Cisco::Node.instance()
 
-version = node.config_get("show_version", "system_image")
-
-node.config_set("vtp", "domain", "mycompany.com")
-```
-
-### Remote usage
-
-#### Low-level Client API
-
-```ruby
-require 'cisco_node_utils'
-
-client = Cisco::Client.create('n3k.mycompany.com', 'username', 'password')
```

Comment:
These examples were outdated and incorrect.  The behavior changed when the cisco_node_utils.yaml file functionality was added.